src/dpp/guild.cpp

```diff
@@ -436,7 +439,7 @@ guild& guild::fill_from_json(discord_client* shard, nlohmann::json* d) {
 	if (d->find("unavailable") == d->end() || (*d)["unavailable"].get<bool>() == false) {
 		/* Clear unavailable flag if set */
 		if (this->flags & dpp::g_unavailable) {
-			this->flags -= dpp::g_unavailable;
```

Comment:
is this a feature or a bug?

Reply:
both do the same thing so long as the value you subtract is a power of two...